src/DataBox/DataBox/Common/Cmdlets/NewDataBoxJobCmdlet.cs

```diff
@@ -0,0 +1,204 @@
+                StreetAddress2 = this.StreetAddress2,
+                StreetAddress3 = this.StreetAddress3,
+                City = this.City,
+                StateOrProvince = this.StateOrProvinceCode,
+                Country = this.CountryCode,
+                PostalCode = this.PostalCode
+            };
+
+            
+
+            ContactDetails contactDetails = new ContactDetails()
+            {
+                Phone = this.PhoneNumber,
+                EmailList = EmailId,
+                ContactName = this.ContactName
+            };
+
+            List<DestinationAccountDetails> destinationAccountDetails = new List<DestinationAccountDetails>();
+
+            foreach (var storageAccount in StorageAccountResourceId)
+            {
+                destinationAccountDetails.Add(
+                new DestinationAccountDetails(storageAccount));
+            }
+
+
+
+            DataBoxDiskJobDetails diskDetails;
+            DataBoxJobDetails databoxDetails;
+            DataBoxHeavyJobDetails heavyDetails;
+
+            JobResource newJobResource = new JobResource();
+
+            Sku sku = new Sku();
+            switch (DataBoxType)
+            {
+                case "DataBoxDisk":
+                    diskDetails = new DataBoxDiskJobDetails(contactDetails, shippingAddress, destinationAccountDetails, expectedDataSizeInTeraBytes: ExpectedDataSizeInTeraBytes);
+                    sku.Name = SkuName.DataBoxDisk;
+                    newJobResource = new JobResource(Location, sku, details: diskDetails);
+                    break;
+                case "DataBox":
+                    databoxDetails = new DataBoxJobDetails(contactDetails, shippingAddress, destinationAccountDetails);
+                    sku.Name = SkuName.DataBox;
+                    newJobResource = new JobResource(Location, sku, details: databoxDetails);
+                    break;
+                case "DataBoxHeavy":
+                    heavyDetails = new DataBoxHeavyJobDetails(contactDetails, shippingAddress, destinationAccountDetails);
+                    sku.Name = SkuName.DataBoxHeavy;
+                    newJobResource = new JobResource(Location, sku, details: heavyDetails);
+                    break;
+                default: break;
+            }
+
+
+            AddressValidationOutput addressValidationResult = ServiceOperationsExtensions.ValidateAddressMethod(
+                DataBoxManagementClient.Service, Location, shippingAddress, newJobResource.Sku.Name);
+
+
+            if (addressValidationResult.ValidationStatus != AddressValidationStatus.Valid)
+            {
+                Console.WriteLine("Address validation status: {0}", addressValidationResult.ValidationStatus);
+
+                //print alternate address
+                if (addressValidationResult.ValidationStatus == AddressValidationStatus.Ambiguous)
+                {
+                    Console.WriteLine("\nSUPPORT ADDRESSES: ");
```

Comment:
Do not use Console.WriteLine in cmdlet code.  If this is meant as a message, use WriteVerbose.  If this is meant to be the ouptut of the cmdlet, you should return an object and you can alter the output format to write the data the way you want it to look.